app/services/channel_promo_registrar.rb

```diff
@@ -0,0 +1,42 @@
+# Register channel for promo
+class ChannelPromoRegistrar < BaseApiClient
+  def initialize(channel:)
```

Comment:
minor: add a TODO note that promo ID is hardcoded, and we should make it a service param later if needed.